cla-backend/cla/routes.py

```diff
@@ -1381,17 +1325,14 @@ def sign_request(provider: hug.types.one_of(get_supported_repository_providers()
 
     The endpoint that will initiate a CLA signature for the user.
     """
-    cla.log.debug('GET /v2/repository-provider/' + str(provider) +
-                  '/sign/' + str(installation_id) + '/' + str(github_repository_id) +
-                  '/' + str(change_request_id))
     return cla.controllers.repository_service.sign_request(provider,
                                                            installation_id,
                                                            github_repository_id,
                                                            change_request_id,
                                                            request)
 
 
-@hug.get('/repository-provider/{provider}/oauth2_redirect', versions=2)
```

Comment:
Was this line meant to be deleted?

Reply:
yes, issue when GH rebased changes from conflict. Let me fix.